src/Orc.Automation/Properties/AssemblyInfo.cs

```diff
@@ -11,8 +11,9 @@
 [assembly: NeutralResourcesLanguage("en-US")]
 
 [assembly: XmlnsPrefix("http://schemas.wildgums.com/orc/automation", "orcautomation")]
+[assembly: XmlnsPrefix("http://schemas.wildgums.com/orc/automation/controls", "orcautomationcontrols")]
 [assembly: XmlnsDefinition("http://schemas.wildgums.com/orc/automation", "Orc.Automation")]
-[assembly: XmlnsDefinition("http://schemas.wildgums.com/orc/automation", "Orc.Automation.Controls")]
+[assembly: XmlnsDefinition("http://schemas.wildgums.com/orc/automation/controls", "Orc.Automation.Controls")]
```

Comment:
do you really think this is a good idea? Isn't it wrong that a control with exactly the same name exists in different namespaces?

Reply:
it's not a control...it's automation part of same control

Reply:
but do we need it to be available as xaml namespace? Maybe it hsould be (sounds stupid, I know): Orc.Automation.Automation (or something like that)?

Reply:
maybe just remove Orc.Automation from xaml namespace? only Controls is important for xaml

Reply:
If the models / etc are not needed, then I agree

Reply:
absolutely no need ... we need only AutomationControl and TestHost (only for test proj) controls... other classes/models only for tests